noetic/distribution.yaml

```diff
@@ -7901,6 +7901,13 @@ repositories:
       url: https://github.com/ros-perception/vision_opencv.git
       version: noetic
     status: maintained
+  visp:
+    release:
```

Comment:
Please add a `source` (and optional `doc`) entry.

Reply:
OK. I just submitted [PR28777](https://github.com/ros/rosdistro/pull/28777) with the requested changes